docs/2.8.0/docs/canton/usermanual/kms/external_key_storage_aws.rst

```diff
@@ -0,0 +1,199 @@
+
+.. _external_key_storage_permissions_aws:
+
+AWS KMS Configuration
+---------------------
+
+To start using this feature we need to first enable a KMS for Canton.
+
+:ref:`--Configure AWS KMS for Canton-- <kms_aws_setup>`
+
+The following IAM permissions are required:
+
+- `kms:CreateKey`
+- `kms:TagResource`
+- `kms:Decrypt`
+- `kms:Sign`
+- `kms:DescribeKey`
+- `kms:GetPublicKey`
+
+When you are using cross-account keys, you do not need the `kms:CreateKey` and `kms:TagResource` permissions.
+
+External Key Storage and Usage Configuration
+--------------------------------------------
+
+External key storage and usage support can be enabled for a new installation (i.e., during the node
+bootstrap) or for an existing deployment.
+**Be aware that if a node has already been deployed you need to** :ref:`perform a node migration <participant_aws_kms_migration>`.
+Simply adding the following configuration is not enough.
+
+In the example below, we configure **a new** Canton participant node (called ``participant1``) to generate and
+store private keys in an external AWS KMS. The same configuration is applicable for all other node entities, e.g. domain-manager,
+mediators, sequencers.
+
+.. literalinclude:: /canton/includes/mirrored/enterprise/app/src/test/resources/aws-kms-provider-tagged.conf
+   :language: none
+   :start-after: user-manual-entry-begin: KmsProviderConfig
+   :end-before: user-manual-entry-end: KmsProviderConfig
+
+An example configuration that puts together both AWS KMS and external storage configuration is shown below:
+
+.. literalinclude:: /canton/includes/mirrored/enterprise/app/src/test/resources/aws-kms-provider-tagged.conf
+   :language: none
+   :start-after: user-manual-entry-begin: AwsKmsProviderFullConfig
+   :end-before: user-manual-entry-end: AwsKmsProviderFullConfig
+
+By default nodes run a ``tink`` provider that is not compatible with KMS provider. If you want to continue to have other nodes
+running a `non-KMS Canton` you are obliged to use a ``jce`` provider
+and you must explicitly configure it to use the KMS supported algorithms as the required algorithms. Here is an
+example for a domain:
+
+.. literalinclude:: /canton/includes/mirrored/enterprise/app/src/test/resources/aws-kms-provider-tagged.conf
+   :language: none
+   :start-after: user-manual-entry-begin: JceProviderDomainConfig
+   :end-before: user-manual-entry-end: JceProviderDomainConfig
+
+And here is an example for a participant:
+
+.. literalinclude:: /canton/includes/mirrored/enterprise/app/src/test/resources/aws-kms-provider-tagged.conf
+   :language: none
+   :start-after: user-manual-entry-begin: JceProviderParticipantConfig
+   :end-before: user-manual-entry-end: JceProviderParticipantConfig
+
+In other words, a node running with a ``kms`` provider is only ever able to communicate with other nodes running
+a ``kms`` or ``jce`` providers.
+
+A KMS only supports the :ref:`following cryptographic schemes <canton_supported_keys>`.
```

Comment:
This links to a table detailing crypto schemes for all providers, not only KMS. So I'd rephrase to 

```suggestion
See :ref:`this table <canton_supported_keys>` for a description of cryptographic schemes supported by the KMS provider.
```